<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI_ThirdPage</class>
 <widget class="QWizardPage" name="UI_ThirdPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <widget class="QListView" name="RowList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>231</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
  </widget>
  <widget class="QListView" name="ItemList">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>211</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddItem">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>190</y>
     <width>161</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelItem">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTool">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Tool</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Application Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Application Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Content Row</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combox_apptype">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_appname">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_content">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>URL Row</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddRow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Row</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelRow">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>500</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Row</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_url">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddMsk">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>500</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Item Mask</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
